contentScript.js

```diff
@@ -28,6 +28,49 @@ function addingCssElement(elementId, status, numRelevantFiles) {
 	}
 };
 
+
+function addCssElementToBitbucket(highlightedPRIds) {
+
+	// To do : remove this setTime out method once data is coming from api 
+	setTimeout(() => {
+		let tables = document.getElementsByTagName('table');
+		tables = tables[0];
+		let allLinks = Array.from(tables.getElementsByTagName('a'));
+		function changingCss(id,status,numRelevantFiles = 1){
+			let backgroundColor = status == 'Important' ? '#ffbab5' : '#f1f549';
+			let tagBackgroundColor = status == 'Important' ? '#e80f00' : 'rgb(164, 167, 0)';
+			allLinks.map((item) => {
+				let link = item.getAttribute('href').split('/');
+				let linkLength = link.length;
+				let prId = link[linkLength - 1]; // getting the last element from url which is pr id. 
+				if (prId == id) {
+					const beforeElement = document.createElement('span');
+					beforeElement.innerText = `${status} (${numRelevantFiles})`;
+					beforeElement.style.display = 'inline-block';
+					beforeElement.style.marginRight = '5px';
+					beforeElement.style.backgroundColor = `${tagBackgroundColor}`;
+					beforeElement.style.color = 'white';
+					beforeElement.style.padding = '2px';
+					beforeElement.style.paddingLeft = '5px';
+					beforeElement.style.paddingRight = '5px'
+					beforeElement.style.borderRadius = '3px';
+					let parent = item.parentNode.parentNode.parentNode.parentNode.parentNode.parentNode.parentNode;
```

Comment:
https://www.geeksforgeeks.org/closest-ancestor-element-that-has-a-specific-class-in-javascript/

Reply:
https://developer.mozilla.org/en-US/docs/Web/API/Element/closest

Reply:
```suggestion
					let parent = item.closest("tr");
```